colossalai/inference/README.md

```diff
@@ -55,18 +55,25 @@ dependencies
 
 ```bash
 pytorch= 1.13.1 (gpu)
+cuda>= 11.6 
 transformers= 4.30.2
 triton==2.0.0.dev20221202
-vllm=
-flash-attention=
+# for install vllm, please use this branch to install https://github.com/tiandiao123/vllm/tree/setup_branch
+vllm
+# for install flash-attention, please use commit hash: 67ae6fd74b4bc99c36b2ce524cf139c35663793c
```

Comment:
Are we going to add cmd like `pip install git+https://github.com/...`?

Reply:
we can

Reply:
add it later on .